src/ames_housing/io_managers/base_lakefs_io_manager.py

```diff
@@ -52,27 +54,29 @@ def get_path(
         str
             Path to the object in lakeFS.
         """
+
+        metadata = get_metadata(context)
+
+        repository = metadata.get("repository")
+
+        asset_path = metadata.get("path") + context.asset_key.path
+        path = "/".join(asset_path)
```

Comment:
Maybe use a `pathlib.Path` here for building the path.

Reply:
Done in 320aeda385bd9c18086cce79ccc2f78e58c67650